app/controllers/incurred_incidentals_controller.rb

```diff
@@ -56,6 +57,20 @@ def update
 
   private
 
+  def upload_documents
+    # only do this on sucess and with a file
+    if @incurred_incidental.errors.empty? && params[:file]
+      # only allow types of uploaded file
+      params.require(:file).transform_values! { |uf| uf if uf.class == ActionDispatch::Http::UploadedFile }
+
+      params.require(:file).to_unsafe_h.map(&:itself).each do |id, uploaded_file|
```

Comment:
Ok I still want to do this part though
![image](https://cloud.githubusercontent.com/assets/2114323/21730191/937fddb8-d41c-11e6-95fe-c0e016c46886.png)